seed/views/properties.py

```diff
@@ -492,21 +740,26 @@ def _get_property_view(self, pk, cycle_pk):
     @api_endpoint_class
     @ajax_request_class
     @detail_route(methods=['GET'])
-    def view(self, request, pk=None):
+    def view(self, pk=None):
         """
         Get the property view
-        ---
-        parameters:
-            - name: cycle_id
-              description: The cycle ID to query on
-              required: true
-              paramType: query
         """
-        cycle_pk = request.query_params.get('cycle_id', None)
-        if not cycle_pk:
-            return JsonResponse(
-                {'status': 'error', 'message': 'Must pass in cycle_id as query parameter'})
-        result = self._get_property_view(pk, cycle_pk)
+        try:
+            property_view = PropertyView.objects.select_related(
```

Comment:
this is a pretty big change... that is, we are now passing in property view IDs instead of the property ids. I am all for this.